### PR TITLE
Bumped Buefy version to 0.9.25

### PR DESCRIPTION
**Pull request by gmercey**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "collectCoverage": true
   },
   "dependencies": {
-    "buefy": "^0.9.23"
+    "buefy": "^0.9.24"
```
Comment (wesdevpro) [inline package.json:33]: The version need to be `v0.9.25`

Comment (gmercey) [reply package.json]: my bad! PR updated.

**Review by wesdevpro — APPROVED** [review]

**Comment by miroslavgeorgiev** [comment]
There is no published version in NPM - still 0.4.25:
https://www.npmjs.com/package/nuxt-buefy?activeTab=versions

**Comment by gmercey** [comment]
I do not believe I have the permission to publish the version to NPM. @wesdevpro maybe ?

**Comment by wesdevpro** [comment]
> I do not believe I have the permission to publish the version to NPM. @wesdevpro maybe ?

I need to create a GitHub action to build then publish `nuxt-buefy` to npm

**Comment by wesdevpro** [comment]
> There is no published version in NPM - still 0.4.25: https://www.npmjs.com/package/nuxt-buefy?activeTab=versions @gmercey

Please see: https://www.npmjs.com/package/nuxt-buefy/v/0.4.25

**Comment by wesdevpro** [comment]
@gmercey We may need to Bump the version to 0.9.26 as there was already a version 0.9.25 that was published 7 months ago

**Comment by wesdevpro** [comment]
@gmercey please see #136

**Pull request merged**